hmda-reporting/src/main/scala/hmda/reporting/api/http/ReportingHttpApi.scala

```diff
@@ -12,44 +11,43 @@ import ch.megard.akka.http.cors.scaladsl.CorsDirectives.{
   corsRejectionHandler
 }
 import de.heikoseeberger.akkahttpcirce.FailFastCirceSupport._
-import hmda.api.http.model.ErrorResponse
 import hmda.model.institution.{
   HmdaFiler,
   HmdaFilerResponse,
   MsaMd,
   MsaMdResponse
 }
 import hmda.query.repository.ModifiedLarRepository
-import hmda.reporting.repository.TsComponent
+import hmda.reporting.repository.InstitutionComponent
 import hmda.util.http.FilingResponseUtils.entityNotPresentResponse
 import io.circe.generic.auto._
 import slick.basic.DatabaseConfig
 import slick.jdbc.JdbcProfile
 
 import scala.concurrent.ExecutionContext
 import scala.util.{Failure, Success}
-trait ReportingHttpApi extends TsComponent {
+trait ReportingHttpApi extends InstitutionComponent {
   implicit val system: ActorSystem
   implicit val materializer: ActorMaterializer
   implicit val ec: ExecutionContext
   implicit val timeout: Timeout
 
-  val tsRepository: TransmittalSheetRepository
+  val institutionRepository: InstitutionRepository
 
   val filerListRoute: Route = {
     path("filers" / IntNumber) { filingYear =>
       get {
 
         val futFilerSet =
-          tsRepository
-            .getAllSheets(filingYear)
+          institutionRepository
+            .getAllFilers()
             .map(
               sheets =>
                 sheets
-                  .map(tsEntity =>
-                    HmdaFiler(tsEntity.lei,
-                              tsEntity.institutionName,
-                              tsEntity.year.toString))
+                  .map(instituionEntity =>
+                    HmdaFiler(instituionEntity.lei,
+                              instituionEntity.respondentName,
+                              "2018"))
```

Comment:
can we use the "activityYear" field in lieu of hardcoding 2018 here?

Reply:
Didn't see that field. Will include.